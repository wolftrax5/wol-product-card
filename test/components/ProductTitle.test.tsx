import React from 'react';
import render from 'react-test-renderer';
import { ProductTitle, ProductCard } from '../../src/components';
import { product1 } from '../data/products';

describe('Product Title', () => {
  test('debe de mostrar el compoennete correctamente con el titlulo Personalizado', () => {
    const wrapper = render.create(
      <ProductTitle title="custome Product" className="title-class" />
    );
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
  test('debe de mostrar el compoennete correctamente con el titlulo Personalizado', () => {
    const wrapper = render.create(
      <ProductCard product={product1}>{() => <ProductTitle />}</ProductCard>
    );
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
