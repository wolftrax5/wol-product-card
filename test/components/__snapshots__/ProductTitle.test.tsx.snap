// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product Title debe de mostrar el compoennete correctamente con el titlulo Personalizado 1`] = `
<span
  className="productDescription title-class"
>
  custome Product
</span>
`;

exports[`Product Title debe de mostrar el compoennete correctamente con el titlulo Personalizado 2`] = `
<div
  className="productCard undefined"
>
  <span
    className="productDescription undefined"
  >
    Coffee mug - sin imagen
  </span>
</div>
`;
